### PR TITLE
Add schedule

### PR DESCRIPTION
**Pull request by RealOrangeOne**
Copied from https://github.com/srobo/tasks/issues/73

Closes https://github.com/srobo/tasks/issues/161

### CODE REVIEW

**Comment by Tyler-Ward** [comment]
Have we confirmed the schedule yet as the schedule ticket is still open?

File: _events/sr2019/competition.md [hunk 1]
```diff
@@ -12,9 +12,32 @@ The Student Robotics Competition marks the ultimate stage of SR2019, showcasing
 
 The competition will be held from *Saturday 6th April* until *Sunday 7th April*. Doors open 9am and we aim to finish around 6pm.
 
-The Saturday will begin with a briefing on the rules and general struture of the weekend, followed by some time for teams to test their robots in the arena. Soon after, the league matches will start, in which your robot will compete against 3 other bots to rise up the league. These continue for the rest of the day.
-
-On Sunday morning, the league matches will continue. Once these have concluded, we will begin knockout matches, the order of which is determined by your position in the league. The remaining teams will compete in the finals, followed by the awards for the winning teams.
```
Comment (PeterJCLaw) [inline _events/sr2019/competition.md]: Any reason not to leave this in place as well as the tables? The prose is somewhat nicer to read (and I suspect somewhat friendlier to both news organisations and screen readers).

Comment (PeterJCLaw) [reply _events/sr2019/competition.md]: (perhaps pull the paragraphs under each of the new headings though)

File: _events/sr2019/competition.md [hunk 2]
```diff
@@ -12,9 +12,32 @@ The Student Robotics Competition marks the ultimate stage of SR2019, showcasing
...
+|-------|-------------------------|
+| 9:00  | Doors open              |
+| 9:30  | Tinker Time             |
+| 10:30 | League Matches          |
+| 12:00 | Lunch and Tinker time   |
+| 13:00 | League Matches          |
+| 14:40 | Knockouts               |
+| 17:00 | Prize Ceremony          |
+| 17:15 | Kit return              |
+| 18:00 | ENd of day              |
```
Comment (PeterJCLaw) [inline _events/sr2019/competition.md]:
```suggestion
| 18:00 | End of day              |
```

File: _events/sr2019/competition.md [hunk 3]
```diff
@@ -12,9 +12,32 @@ The Student Robotics Competition marks the ultimate stage of SR2019, showcasing
...
+### Day 1 - 6th April, 2019
+
+
+| Time  | Activity                         |
+|-------|----------------------------------|
+| 9:00  | Doors open                       |
+| 9:30  | Introduction and safety briefing |
+| 9:40  | Tinker time                      |
+| 11:00 | League matches                   |
+| 12:00 | Lunch and Tinker time            |
```
Comment (PeterJCLaw) [inline _events/sr2019/competition.md]: We're inconsistent about our capitalisation here and elsewhere. I suggest making all of these "Sentence case"

**Comment by RealOrangeOne** [comment]
> Have we confirmed the schedule yet as the schedule ticket is still open?

I'd forgotten about this, but the only thing outstanding is the group photo, which i've added a comment to in https://github.com/srobo/tasks/issues/73#issuecomment-473600934

**Pull request merged**